src/utils/interpolateBorder.ts

```diff
@@ -51,7 +53,11 @@ function getInterpolateConstant(value: number): BorderInterpolationFunction {
     const newColumn = interpolateConstantPoint(column, image.width);
     const newRow = interpolateConstantPoint(row, image.height);
     if (newColumn === -1 || newRow === -1) {
-      return value;
+      if (!Array.isArray(value)) {
```

Comment:
I'm worried about performance with this new if-else here. It will be called into 3 nested loops (rows, columns, channels).

Maybe this if-else statement should be higher to return independent interpolation functions.

What do you think @stropitek ?